CMakeModules/CompilerSettings.cmake

```diff
@@ -77,11 +77,19 @@ if((CMAKE_COMPILER_IS_GNUCXX OR IS_ICPC) AND NOT MINGW)
     endif()
 endif()
 
+# By default CMake sets RPATH for binaries in the build tree, but clears
+# it when installing. Switch this option off if the default behaviour is
+# desired.
+option(NO_DEFAULT_RPATH "Set RPATH for installed binaries" ON)
```

Comment:
All FCL CMake options that are not global CMake options (such as `BUILD_SHARED_LIBS` or `BUILD_TESTING`) are prefixed with `FCL_`, I suggest to use the same convention here. Furthermore given that this is a rather advanced option, I suggest marking it as advanced using [`mark_as_advanced`](https://cmake.org/cmake/help/v3.0/command/mark_as_advanced.html).